splitwise/src/com/test/services/SplitWiseService.java

```diff
@@ -0,0 +1,113 @@
+package com.test.services;
+import com.test.models.User;
+
+import java.util.*;
+
+public class SplitWiseService {
+    Map<String,User> usersMap=new HashMap<String,User>();
+    HashMap<User,HashMap<User,Integer>> expenseTracker=new HashMap<User,HashMap<User,Integer>>();
+
+    public void setUsers(List<User> users) {
+        for(User user:users)
+        {
+            usersMap.put(user.getUid(),user);
+            expenseTracker.put(user,new HashMap<User,Integer>());
+        }
+    }
+
+    public void splitBalance(String command) {
+        String[] expensedata=command.split(" ");
```

Comment:
Should keep input parsing outside of core logic

Reply:
Your execute command would have been a good candidate for parsing and passing an object here.